flow360/component/simulation/meshing_param/params.py

```diff
@@ -74,6 +72,34 @@ class MeshingParams(Flow360BaseModel):
         description="Additional fine-tunning for refinements.",
     )
     # Will add more to the Union
-    volume_zones: List[RotationCylinder] = pd.Field(
-        default=[], description="Creation of new volume zones."
+    volume_zones: Optional[List[Union[RotationCylinder, AutomatedFarfield]]] = pd.Field(
+        default=None, description="Creation of new volume zones."
     )
+
+    @pd.field_validator("volume_zones", mode="after")
+    @classmethod
+    def _finalize_automated_farfield(cls, v) -> Self:
+        if v is None:
+            # User did not put anything in volume_zones so may not want to use volume meshing
+            return v
```

Comment:
I would assume that if you don't want to use meshing, entire object "meshing" should be None. If meshing exists, I would assume it is indented to be used

Reply:
Yes but if the user only wants the surface meshing then the volume_zones can be None.